code/API_definitions/location-verification.yaml

```diff
@@ -6,21 +6,25 @@ info:
 
     # Introduction
 
-    Customers are able to verify whether the location of certain user device is within the area specified. Currently the only area supported is a circle determined by the provided coordinates (latitude and longitude) and some expected accuracy (radius).
+    Customers are able to verify whether the location of certain user device is within the area specified. Currently the only area supported as input is a circle determined by the a set of coordinates (latitude and longitude) and some expected accuracy (radius).
 
     The verification result depends on the network's ability and accuracy to locate the device at the requested area. 
     
-    * The network locates the device within the requested area, the verification result is `TRUE`.
-    * The requested area may not match the area where the network locates the device. In this case, the verification result is `FALSE` . 
-    * The requested area partially match the area where the network locates the device, the verification result is `PARTIAL`. In this case, a `match_rate` could be included in the response, indicating an estimation of the likelihood of the match in percent.
-    * Lastly, the network may not be able to locate the device. In this case, the verification result is `UNKNOWN`
+    * If the network's estimation of the device's location is fully contained within the requested area, the verification result is `TRUE`.
+    * If the network's estimation of the device's location does not overlap with the requested area at all, the verification result is `FALSE` . 
+    * If the network's estimation of the device's location partially overlaps with the requested area, or it fully contains the requested area (because is larger), the result is 'PARTIAL'. In this case, a `match_rate` may be included in the response, indicating an estimation of the likelihood of the match in percent.
```

Comment:
```suggestion
    * If the network's estimation of the device's location partially overlaps with the requested area, or it fully contains the requested area (because it is larger), the result is 'PARTIAL'. In this case, a `match_rate` may be included in the response, indicating an estimation of the likelihood of the match in percent.
```